melange.opam

```diff
@@ -13,6 +13,7 @@ depends: [
   "cmdliner" {>= "1.1.0"}
   "base64" {>= "3.1.0"}
   "cppo" {build}
+  "reason" {with-test}
```

Comment:
this needs to be written in the `dune-project` file

Reply:
done in https://github.com/melange-re/melange/pull/503/commits/d0d7a3f834b850f42890034e2874fdd1c0596229.

Reply:
(i don't like how opam files are generated by dune but they are also tracked by git for publishing purposes, so if I change opam i can see git showing changes and I assume opam is the source, it always confuses me)